applications/machine_learning/README.rst

```diff
@@ -79,6 +79,31 @@ By default, the application uses pretrained machine leaning models deployed in `
 The application displays LED effects that correspond to the machine learning results.
 For more detailed information, see the `User interface`_ section.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the Machine Learning application modules can be suspended or turned off if they are not in use.
+The application uses :ref:`caf_power_manager` module for that purpose.
+It means that Zephyr power management is used when device is in active on suspended state,
+but power off state is forced directly by :ref:`caf_power_manager` as Zephyr's PM_STATE_SOFT_OFF.
+
+* In the :c:enumerator:`POWER_MANAGER_LEVEL_SUSPENDED` state, the device maintains the active Bluetooth connection.
+* In the :c:enumerator:`POWER_MANAGER_LEVEL_OFF` state, the CPU is switched to the off mode.
+
+In both cases, most of the functionalities are disabled.
+For example, LEDs and sensors are turned off and Bluetooth advertising is stopped.
+
+Any button press can wake up the device.
+
+For the Thingy:53 sensor supports trigger that can be used for active power management.
+As long as device acceleration is detected the board is kept in active state.
+When the board is in :c:enumerator:`POWER_MANAGER_LEVEL_SUSPENDED` state it can be woken up by acceleration threshold - it means by moving the device.
```

Comment:
```suggestion
For the Thingy:53, the sensor supports a trigger that can be used for active power management.
As long as the device detects acceleration, the board is kept in the active state.
When the board is in :c:enumerator:`POWER_MANAGER_LEVEL_SUSPENDED` state, it can be woken up by acceleration threshold, that is by moving the device.
```